test/verifiedTests/testFVA/testFVA.m

```diff
@@ -24,7 +25,7 @@
 tol = 1e-8;
 
 % define the solver packages to be used to run this test
-solverPkgs = {'tomlab_cplex', 'glpk'};
+solverPkgs = {'tomlab_cplex'};
```

Comment:
fvamin/fvamax outputs with minOrigSol option requires a QP which is not provided with glpk.